### PR TITLE
v18.3 beta

### PR DESCRIPTION
**Pull request by steveseguin**
fixed fade-ins and added transitions, plus a group of bug fixes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -177,7 +176,7 @@
 		<span
 				id="helpbutton"
 				title="Show Help Info" 
-				onclick="warnUser('For support, please browse https://reddit.com/r/obsninja or join the live chat on Discord at https://discord.obs.ninja.\n\nThe Docs also contains many help guides and advanced settings, located at https://docs.vdo.ninja.\n\nTo access the video stats menu, hold CTRL (command) and Left-Click on a video. Most video issues can be fixed by using Wired Internet instead of Wi-Fi.')"
+				onclick="warnUser('For support, please browse https://reddit.com/r/obsninja or join the live chat on Discord at https://discord.vdo.ninja.\n\nThe Docs also contains many help guides and advanced settings, located at https://docs.vdo.ninja.\n\nTo access the video stats menu, hold CTRL (command) and Left-Click on a video. Most video issues can be fixed by using Wired Internet instead of Wi-Fi.')"
```
Comment (jcalado) [inline index.html:179]: should `https://reddit.com/r/obsninja` be replaced with `https://reddit.com/r/vdoninja`?

File: electron.html [hunk 2]
```diff
@@ -364,11 +364,9 @@
 
 var urlParams = new URLSearchParams(window.location.search);
 
-if (navigator.userAgent.indexOf('Mac OS X') != -1){
-	document.getElementById("warning4mac").style.display="block";
-} else if (location.hostname.toLowerCase() == "obs.ninja"){
+if ((location.hostname.toLowerCase() == "vdo.ninja") || (location.hostname.toLowerCase() == "obs.ninja")){
 	try {
-		if (navigator.userAgent.toLowerCase().indexOf(' electron/') > -1) { // for now, just PC or Linux versions only.
+		if (navigator.userAgent.toLowerCase().indexOf(' electron/') > -1) {
```
Comment (filiptronicek) [inline electron.html:369]: I think we could use `.includes()` here.
```suggestion
		if (navigator.userAgent.toLowerCase().includes(' electron/')) { 
```

File: electron.html [hunk 3]
```diff
@@ -364,11 +364,9 @@
 
 var urlParams = new URLSearchParams(window.location.search);
 
-if (navigator.userAgent.indexOf('Mac OS X') != -1){
-	document.getElementById("warning4mac").style.display="block";
-} else if (location.hostname.toLowerCase() == "obs.ninja"){
+if ((location.hostname.toLowerCase() == "vdo.ninja") || (location.hostname.toLowerCase() == "obs.ninja")){
```
Comment (filiptronicek) [inline electron.html:367]: Using three equal sings will make this more reliable

```suggestion
if ((location.hostname.toLowerCase() === "vdo.ninja") || (location.hostname.toLowerCase() === "obs.ninja")){
```

File: convert.html [hunk 4]
```diff
@@ -28,6 +28,14 @@
 </head>
 <body style='color:white'>
 
+<script>
+
+if (window.location.hostname.indexOf("vdo.ninja") == 0) {
```
Comment (filiptronicek) [inline convert.html:33]: Here, [eqeqeeq ](https://eslint.org/docs/rules/eqeqeq)should apply

```suggestion
if (window.location.hostname.indexOf("vdo.ninja") === 0) {

```

**Review by filiptronicek — CHANGES_REQUESTED** [review]
👀

**Pull request merged**